src/components/Popover/index.tsx

```diff
@@ -99,18 +99,15 @@ export default function Popover({
   const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(null)
   const [arrowElement, setArrowElement] = useState<HTMLDivElement | null>(null)
 
-  const options = useMemo(
-    (): Options => ({
-      placement,
-      strategy: 'fixed',
-      modifiers: [
-        { name: 'offset', options: { offset: [offsetX, offsetY] } },
-        { name: 'arrow', options: { element: arrowElement } },
-        { name: 'preventOverflow', options: { padding: 8 } },
-      ],
-    }),
-    [arrowElement, offsetX, offsetY, placement]
-  )
+  const options: Options = {
```

Comment:
oh i feel like you may want to still use the `memo` here - are u sure `usePopper` doesn't care about the new object every time?

Reply:
My bad, double checked in the docs! https://popper.js.org/react-popper/v2/faq/#why-i-get-render-loop-whenever-i-put-a-function-inside-the-popper-configuration